decentraland/unity-explorer
# Feat: worlds compatibility chat commands

Added chat commands:
- `/goto example.dcl.eth`
- `/world example.dcl.eth`
- `/goto genesis`
- `/goto X,Y`

going from world to coordinates also change to the main realm. 
Pressing JumpIn button from the Map when in the World, also changes realm to the main one.

# How to test
1.
- type in chat following commands: `/goto olavra.dcl.eth` or  `/goto yemel.dcl.eth` or `/world olavra.dcl.eth`  or  `/world yemel.dcl.eth` 
- [expected]: you should see loading screen and then appear in the respective world

2.
- type `/goto 100,100` or any coordinate (from any World or Realm) 
- [expected]: you should appear in the Genesis City in that coordinates

3. 
- press Jump-In button from the Map when you in the World
- [expected]: you should appear in Genesis City in that place
